doc/developer-guide/cripts/cripts-misc.en.rst

```diff
@@ -243,12 +321,14 @@ Using the ``UUID`` object is simple, via the ``Get()`` method. Here's an example
 
    do_remap()
    {
-     static borrow req = cripts::Client::Request::Get();
+     borrow resp = cripts::Client::Response::Get();
```

Comment:
Interesting ... I would have thought that the Client Response wouldn't yet be available in do_remap. Clearly you thought about it to change to it from req, but, just wanted to double check.